Server/fn_findSpawnSegment.sqf

```diff
@@ -6,7 +6,7 @@ params ["_currentInstanceIndex", "_allPlayerPositions", "_minSpawnDistance", "_m
 
 private _spawnDistanceDiff = _maxSpawnDistance - _minSpawnDistance;
 private _roadSegment = "NULL";
-private _refPlayerPos = (_allPlayerPositions select (floor (random (count _allPlayerPositions))));
+private _refPlayerPos = (_allPlayerPositions selectRandom _allPlayerPositions);

```

Comment:
Nooo not like that
https://community.bistudio.com/wiki/selectRandom
I didn't expect you to just take what I said 1 to 1 :D